lib/msf/core/module/target.rb

```diff
@@ -241,6 +241,14 @@ def payload_encoder
     opts['Payload'] ? opts['Payload']['Encoder'] : nil
   end
 
+  #
+  # The payload NOP generator or generators that can be used when generating the
+  # encoded payload (such as x86/opty2 and so on).
+  #
+  def payload_nop
+    opts['Nop'] ? opts['Payload']['Nop'] : nil
```

Comment:
do you mean `opts['Payload'] ? opts['Payload']['Nop']` here?

Reply:
I do. This is why it's not working right now, lol.

Reply:
Fixed. I was in `pry` on that line when you commented. Review and testing ftw!